src/test/java/org/fundacionjala/coding/german/EANValidatorTest.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.coding.german;
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Administrator on 8/29/2017.
+ */
+public class EANValidatorTest {
```

Comment:
to increase the code coverage you should test when the checksum is 0.
for instance.
```java
 assertTrue(eanValidatorTest.validate("9783827317100"));
```